### PR TITLE
Reduce Test Flakiness

### PR DESCRIPTION
**Pull request by Alton09**
## Description

Reduced test flakiness and fixed a couple small UI regressions after implementing #157.

## Validation

- Passed CI

## Submission Checklist
 - [x] The source has been evaluated for semantic versioning changes and are reflected in ```versionName``` under `app/build.gradle`
 - [x] The `CHANGELOG.md` reflects any **feature**, **bug fixes**, or **known issues** made in the source code

### CODE REVIEW

**Review by kbagchiGWC — APPROVED** [review]

**Comment by kbagchiGWC** [comment]
@Alton09 https://app.circleci.com/pipelines/github/twilio/twilio-video-app-android/1306/workflows/4ee3fc3d-4309-45fd-957e-5a57357ce079/jobs/4632 should this be green with the retry?

File: app/src/main/java/com/twilio/video/app/ui/room/RoomViewState.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ data class RoomViewState(
     val isCameraEnabled: Boolean = false,
     val localVideoTrack: VideoTrackViewState? = null,
     val isMicEnabled: Boolean = false,
-    val isAudioMuted: Boolean = true,
```
Comment (kbagchiGWC) [inline app/src/main/java/com/twilio/video/app/ui/room/RoomViewState.kt:20]: Seems like this changes the default. Why was this change needed?

Comment (Alton09) [reply app/src/main/java/com/twilio/video/app/ui/room/RoomViewState.kt:20]: So what this does is when the user starts the app and enters the lobby screen, their audio should not be muted by default.

Comment (kbagchiGWC) [reply app/src/main/java/com/twilio/video/app/ui/room/RoomViewState.kt:20]: Is that how it is for JS and iOS? For all the video apps that we use the default behavior is audio and video are disabled by default. If this change is not required for fixing tests, I would prefer it to be discussed first with @aaalaniz and @donaltoomey.

Comment (Alton09) [reply app/src/main/java/com/twilio/video/app/ui/room/RoomViewState.kt:20]: Oh so this is how it has always been in the Android video app if permissions are enabled. The new JS app does the same. This is a fix as it was regressed in the decoupling story a few PRs back.

Comment (kbagchiGWC) [reply app/src/main/java/com/twilio/video/app/ui/room/RoomViewState.kt:20]: Cool then 👍

**Review by kbagchiGWC — APPROVED** [review]
